.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,8 @@
+blank_issues_enabled: true
+contact_links:
+  - name: Visit the Website
+    url: https://usdx.eu/format/
+    about: The website contains a high-level overview over the song format. If you are new here it's best to start there.
+  - name: UltraStar Discord Server
+    url: https://discord.gg/NS3m3ad
```

Comment:
Seems this is the wrong discord url.. it seemed too familiar (previous version had also references towards performous)